src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -405,6 +413,10 @@
     <Reference Include="System.Net.NameResolution" />
     <Reference Include="System.Diagnostics.Tracing" />
   </ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(OSGroup)' != 'AnyOS' AND '$(TargetGroup)' == 'uap'">
```

Comment:
Right, in this case actually OSGroup will be set to `Windows_NT` everytime that the TargetGroup is uap. Same for `IsPartialFacadeAssembly` as in my other comments.